cmd/sansshell-server/default-policy.rego

```diff
@@ -36,8 +36,8 @@ allow {
 
 allow {
 	input.type = "Exec.ExecRequest"
-	input.message.command = "echo"
-	input.message.args = ["hello", "world"]
+	# input.message.command = "/bin/echo"
```

Comment:
is this an intentional edit?

Reply:
Oops, the echo->/bin/echo part was intentional (exec requires absolute filepath) but the rest was leftover from manual testing. Fixed.